src/ProjectManager.cpp

```diff
@@ -308,16 +307,8 @@ void InitProjectWindow( ProjectWindow &window )
    }
 
    // Lay it out
-   pPage->SetAutoLayout(true);
-   pPage->Layout();
-
-#ifdef EXPERIMENTAL_NOTEBOOK
-   AddPages(this, Factory, pNotebook);
-#endif
-
-   auto mainPanel = window.GetMainPanel();
-
-   mainPanel->Layout();
+   window.GetTrackListWindow()->SetAutoLayout(true);
+   window.GetTrackListWindow()->Layout();
```

Comment:
It would look a little nicer to me, to use a variable to eliminate the repeated calls to `window.GetTrackListWindow()` and `window.GetContainerWindow()`

Reply:
Ok